index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Tic Tac Toe</title>
+    <script src="script.js"></script>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <main>
+    <script src="script.js"></script>
+    <div class=board>
+        <div class= "row1">
+          <div id="cell1" onclick="changeText1()"><h1 id="i1" class="xo"></h1></div>
```

Comment:
As we talked about in our 1-on-1, we want to avoid using the `onclick` in our HTML. Better to use `addEventListener` in our JS files.